framework/auth/views.py

```diff
@@ -196,9 +221,12 @@ def confirm_email_get(token, auth=None, **kwargs):
     user = User.load(kwargs['uid'])
     is_merge = 'confirm_merge' in request.args
     is_initial_confirmation = not user.date_confirmed
-
+    existing_user = request.args.get('existing_user', None)
```

Comment:
As suggested in a previous comment, it doesn't appear necessary to pass `existing_user` anymore. If you need a query param to logout the user, then you could just accept a bool, `?logout=1`